hr_applicant/security/ir.model.access.csv

```diff
@@ -0,0 +1,18 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_applicant_education,applicant.education,model_applicant_education,base.group_system,1,1,1,1
```

Comment:
every user having 1111 access??
We should manage the access, like user cannot delete 1110 and office can have delete access 1111
@ShwetaSerpentCS